### PR TITLE
Build missions

### LINKED ISSUE #4: View: Mission Control (Child)
A place for a child to view all past and any upcoming/current missions, and where the call to action to begin a mission exists.

Necessary Components:
- mission
- agent sidebar (?)
- header/navbar  

Screen Shot 2021-02-19 at 11 03 28 AM

### LINKED ISSUE #11: Component: Mission (Child)
A Mission component will be viewed inside of view: mission control (issue #4).

A mission component could have at least 3 states:
- submitted
- in progress
- new

A mission component indicates the types of tasks it has and other details with indicative
 badges (high level).

What differentiates a mission's state could be indicated by simple visual cue for MVP, later bringing in some type of "mission review"

wireframe ideas:
Screen Shot 2021-02-19 at 11 59 11 AM
Screen Shot 2021-02-19 at 11 59 22 AM
Screen Shot 2021-02-19 at 12 00 27 PM

### PR DESCRIPTION
**Pull request by brabbuss**
### What does this pull request do?
- Add new container and badge components in the UI folder
- Add new `RoundButton` to the UI components
- Add new `Mission` component to populate `MissionControl` view
- Tie FE views into the API
- Add SVG assets
- Add `React.memo()` to exports for optimization 
- Add date parsing function as its own module 

### Issues
closes #4 
closes #11 

### Next steps
- `Mission` needs a completed and incomplete state to render
- Need to work on responsive design if time allows

### Screenshots / Gifs

Screen Shot 2021-02-23 at 3 41 58 PM
Screen Shot 2021-02-23 at 3 41 50 PM

### CODE REVIEW

File: src/assets/art.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+export const art = (
```
Comment (lo-la-do-li) [inline src/assets/art.js:1]: Yayyyy icons! This is exciting!

File: src/modules/mission/Mission.jsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    cursor: "pointer",
+    transition: "transform ease .3s",
+    "&:hover": {
+      transform: "scale(1.05)",
+    },
+  },
+}));
+
+// it's strange, had to pull props out of props??
+const Mission = (props) => {
```
Comment (lo-la-do-li) [inline src/modules/mission/Mission.jsx:82]: This looks awesome, Scotty. Nice work!

**Review by lo-la-do-li — APPROVED** [review]

File: src/modules/mission/Mission.jsx (hunk 2 above)
Comment (brabbuss) [reply src/modules/mission/Mission.jsx:82]: thanks lola ;)

**Review by baileydunning — APPROVED** [review]
This looks great, Scott. Awesome job on all this!

**Pull request merged**